darwin/importer/importer.py

```diff
@@ -318,12 +352,104 @@ def _update_payload_with_properties(
             annotation["annotation_properties"] = dict(_map)
 
 
+def _update_payload_with_item_level_properties(
```

Comment:
Lets rename this and update the description unless we are updating the payload like we do in `_update_payload_with_properties`, since its not updating the payload